scripts/update_webhook.rb

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env ruby
+
+require 'octokit'
+require 'yaml'
+
+if ENV['GITHUB_AUTH_TOKEN'].nil?
+  abort("GITHUB_AUTH_TOKEN is needed to access the GitHub API")
+end
+
+repos = YAML.load_file('../config/repos.yaml').keys
+
+Octokit.configure do |c|
+  c.access_token = ENV['GITHUB_AUTH_TOKEN']
+end
+
+new_url = "http://prprocessor.theforeman.org/pull_request"
```

Comment:
Can we use the HTTPS endpoint?

Reply:
Good question -- I don't actually know. But I can test it.